src/app/core/context/ProjectContext/ProjectContext.tsx

```diff
@@ -0,0 +1,125 @@
+import React, {
```

Comment:
@subject026 with this PR i am adding a new ProjectContext for easy fetching of project-related data -- so far, how much BREAD and Voting Power they have. Curious for your feedback on this piece in particular, if you have any suggestions for improvement or if it looks good